website/docs/components/button/partials/code/component-api.md

```diff
@@ -9,7 +9,7 @@
   <C.Property @name="icon" @type="string">
     Used to show an icon. Any [icon](/icons/library) name is accepted.
     <br/><br/>
-    Tertiary buttons have transparent backgrounds, and interactive elements must communicate interactivity with more than just color. Therefore, a leading or trailing icon is required when using the `tertiary` color. [WCAG 2.1 Criterion 1.4.1: Use of Color (Level A)](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=141#use-of-color)
+    Tertiary buttons have transparent backgrounds, and interactive elements must communicate interactivity with more than just color. Therefore, a leading or trailing icon is required when using the `tertiary` color. [WCAG 2.2 Criterion 1.4.1: Use of Color (Level A)](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=141#use-of-color)
```

Comment:
Should the links like this, that reference in the URL `WAI/WCAG21` be also updated to `WAI/WCAG22`?